firebase-app-distribution/src/main/AndroidManifest.xml

```diff
@@ -44,12 +44,12 @@
 
         <provider
             android:name="androidx.core.content.FileProvider"
-            android:authorities="${applicationId}.provider"
+            android:authorities="${applicationId}.appdistro.fileprovider"
             android:exported="false"
             android:grantUriPermissions="true">
             <meta-data
                 android:name="android.support.FILE_PROVIDER_PATHS"
-                android:resource="@xml/provider_paths" />
+                android:resource="@xml/appdistro_provider_paths" />
```

Comment:
same thing here, I think this should be `firebase_app_distribution_provider_path`